app/core/ui/src/main/kotlin/com/hedvig/android/core/ui/insurance/DocumentRow.kt

```diff
@@ -0,0 +1,71 @@
+package com.hedvig.android.core.ui.insurance
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.hedvig.android.core.designsystem.newtheme.SquircleShape
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.android.core.icons.Hedvig
+import com.hedvig.android.core.icons.hedvig.small.hedvig.ArrowNorthEast
+
+@Composable
+fun DocumentRow(
+  modifier: Modifier = Modifier,
+  title: String,
+  subTitle: String,
+  onClick: () -> Unit,
+) {
+  Row(
+    verticalAlignment = Alignment.CenterVertically,
+    horizontalArrangement = Arrangement.SpaceBetween,
+    modifier = modifier
+      .background(
+        shape = SquircleShape,
+        color = MaterialTheme.colorScheme.background
+      )
+      .fillMaxWidth()
+      .padding(horizontal = 16.dp, vertical = 12.dp)
+      .clickable { onClick() },
+  ) {
+    Column {
+      Row {
+        Text(title)
+        Spacer(modifier = Modifier.padding(1.dp))
+        Text("PDF", fontSize = 11.sp)
+      }
+      Text(subTitle, color = MaterialTheme.colorScheme.secondary)
+    }
+
+    Icon(
+      imageVector = Hedvig.ArrowNorthEast,
+      contentDescription = "link",
+    )
+  }
+}
+
+@Composable
+@HedvigPreview
+fun PreviewDocumentRow() {
+  HedvigTheme {
```

Comment:
With previews it's easier to work with when you already have the surface (and as a result the content colors too) by using a surface composable top level.

Recently I've made a Live Template which looks like this

<img width="1694" alt="image" src="https://github.com/HedvigInsurance/android/assets/44558292/f1f5f476-4476-4fe1-8351-6720c298ae6f">

The text there is:
```
@com.hedvig.android.core.designsystem.preview.HedvigPreview
@androidx.compose.runtime.Composable
private fun Preview$NAME$() {
  com.hedvig.android.core.designsystem.theme.HedvigTheme {
    androidx.compose.material3.Surface(color = androidx.compose.material3.MaterialTheme.colorScheme.background) {
        $NAME$($END$)
    }
  }
}
```

And this makes it so that if you type "prev" and then "enter" it auto-fills the preview for you with some sane defaults.

And for this transition period, you may also want to do `HedvigTheme(useNewColorScheme = true) {` instead, so that you get the new color changes in your previews too.

Reply:
Wow that is so good. I love live templates.